assignments/callbacks.js

```diff
@@ -27,19 +27,32 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  cb(arr.length);
 }
+getLength(items, (length) => {
+  alert(length);
+});
 
 function last(arr, cb) {
```

Comment:
function last(arr, cb) {
  // last passes the last item of the array into the callback.
  cb(arr[arr.length - 1]);
}

last(items, function(lastItem) {
  console.log(lastItem);
});

function sumNums(x, y, cb) {
  // sumNums adds two numbers (x, y) and passes the result to the callback.
  cb(x + y);
}

sumNums(10, 15, function(total) {
   console.log(total);
});

function multiplyNums(x, y, cb) {
  // multiplyNums multiplies two numbers and passes the result to the callback.
   cb(x * y);
}

multiplyNums(10, 15, function(total) {
   console.log(total);
});